### DISCUSSION TITLE
How to disable proxy in docker container?

### DISCUSSION THREAD

**Discussion by lokize** [coder/code-server#6521] [Q&A]
Hi, i have code-server running in a container with this composer:

`version: "2.1"
services:
  code-server:
    image: lscr.io/linuxserver/code-server:latest
    container_name: code-server
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/UTC
      - PASSWORD= #optional
      - HASHED_PASSWORD= #optional
      - SUDO_PASSWORD= #optional
      - SUDO_PASSWORD_HASH= #optional
      #- PROXY_DOMAIN= #optional
      - CS_DISABLE_PROXY=TRUE
      - DEFAULT_WORKSPACE=/config/workspace #optional
    volumes:
      - ./vscode:/config
    ports:
      - 8443:8443
    restart: unless-stopped`

I want disable proxy because i cant open my projects in browser, its opening in : http://localhost:8443/proxy/4001/ and i cant run my application.

**Comment by code-asher** [comment]
Right now disabling the proxy still rewrites the URL, I think you can set `VSCODE_PROXY_URI=` to prevent this for now, but we should fix it (see https://github.com/coder/code-server/issues/6504).
